src/controllers/update/update.filing.date.controller.ts

```diff
@@ -56,3 +57,20 @@ export const post = async(req: Request, res: Response, next: NextFunction) => {
     next(error);
   }
 };
+
+const getFilingDate = async (req: Request, appData: ApplicationData): Promise<{} | undefined> => {
+  // use date stored in appData if present
```

Comment:
I'm just curious as to when the date will be already present in `appData` and when not? Might be worth extending the comment to explain.

Reply:
updated with better comments